keyboards/projectkb/alice/alice.c

```diff
@@ -0,0 +1,27 @@
+#include "alice.h"
+
+void matrix_init_board(void){
+  palSetPadMode(GPIOA, 0, PAL_MODE_OUTPUT_PUSHPULL);
```

Comment:
Would you mind converting these to the GPIO commands, if possible?
https://docs.qmk.fm/#/internals_gpio_control

https://github.com/qmk/qmk_firmware/blob/master/quantum/quantum.h#L166-L184